src/main/java/christmas/domain/Gift.java

```diff
@@ -0,0 +1,34 @@
+package christmas.domain;
+
+import christmas.domain.plan.Menu;
+
+public class Gift {
+    private final static int GIFT_OFFER_LEAST_PRICE = 120000;
```

Comment:
정말 사소한 부분이지만, 숫자의 자릿수가 많아질 때는 이렇게 `_`를 사용해 자리수 구분이 가능합니다!

```suggestion
    private final static int GIFT_OFFER_LEAST_PRICE = 120_000;
```